packages/domain-picker/src/domain-picker/domain-i-own-item.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { __ } from '@wordpress/i18n';
+import { Icon, chevronRight } from '@wordpress/icons';
+import { createInterpolateElement } from '@wordpress/element';
+import { useLocalizeUrl } from '@automattic/i18n-utils';
+
+/**
+ * Internal dependencies
+ */
+
+interface Props {
+	siteSlug: string;
+	source: string;
+}
+
+const DomainIOwn: React.FunctionComponent< Props > = ( { siteSlug, source } ) => {
+	const localizeUrl = useLocalizeUrl();
+
+	return (
+		<label className="domain-picker__suggestion-item contains-link">
+			<div className="domain-picker__suggestion-item-name">
+				<span className="domain-picker__domain-wrapper with-margin">
+					{ createInterpolateElement(
+						__( '<strong>Already own a domain?</strong>', __i18n_text_domain__ ),
+						{
+							strong: <strong />,
+						}
+					) }
+				</span>
+				<div>
+					<span className="domain-picker__item-tip">
+						{ __( "You can use it as your site's address", __i18n_text_domain__ ) }
+					</span>
+				</div>
+			</div>
+			<div className="domain-picker__action-link with-underline">
```

Comment:
`with-underline` class doesn't include the namespace. See https://github.com/Automattic/wp-calypso/pull/47062#discussion_r517293404
```suggestion
			<div className="domain-picker__action-link domain-picker__action-link--with-underline">
```

Reply:
Yep this would be preferred but didn't realise we had agreed on a convention.